list.go

```diff
@@ -65,3 +65,9 @@ func (l *List) Actions() (actions []Action, err error) {
 	}
 	return
 }
+
+// AddCard is just a wrapper for Client.NewCard
+func (l *List) AddCard(card Card) (*Card, error) {
```

Comment:
I thought of doing it this way because just to get the `List` struct we need another HTTP request, and (I imagine) in most cases no one actually needs data from lists, so we're wasting an HTTP request.

But that's ok, makes sense.